Makefile

```diff
@@ -205,11 +205,18 @@ endif
 # ------------
 
 .PHONY: verify-deps
-verify-deps:
+verify-deps: verify-go-deps verify-src-deps
+
+.PHONY: verify-go-deps
+verify-go-deps:
 	go mod verify
 	go mod tidy
 	@git diff --exit-code -- go.mod go.sum
 
+.PHONY: verify-src-deps
```

Comment:
I think just having this verify focus on generated files be clearer. What do you think about changing the name to `verify-generated` ?

Reply:
sounds good